src/Form/FormProtector.php

```diff
@@ -0,0 +1,563 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Form;
+
+use Cake\Core\Configure;
+use Cake\Utility\Hash;
+use Cake\Utility\Security;
```

Comment:
Does this add new dependencies to cake/form?

Reply:
Nope, as validation https://github.com/cakephp/validation/blob/master/composer.json#L27 which already is required.

Reply:
As @dereuromark pointed out it doesn't. Though I doubt anyone uses the `Form` package as standalone anyway.